module/account/src/Domain/Entity/User.php

```diff
@@ -241,6 +258,18 @@ public function changePassword(Password $password): void
         $this->apply(new UserPasswordChangedEvent($password));
     }
 
+    /**
+     * @param bool $isActive
+     *
+     * @throws \Exception
+     */
+    public function checkActivity(bool $isActive): void
```

Comment:
This is Domain object shoud have seperate method to active and deactive user that throw exception when you whant active alredy Actived user or deactive Deactiveted